course_catalog/serializers.py

```diff
@@ -43,64 +42,4 @@ class CourseSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Course
-        # fields = "__all__"
         exclude = ('raw_json',)
```

Comment:
Replace `exclude = ('raw_json',)` with this, otherwise the `raw_json` field is always set to `None` in the model:
```python
        fields = "__all__"
        extra_kwargs = {
            'raw_json': {'write_only': True}
        }
```